taxi_service/urls.py

```diff
@@ -21,5 +21,6 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("", include("taxi.urls", namespace="taxi")),
+    path("taxi/", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("accounts.urls", namespace="accounts"))
```

Comment:
You shoul provide authentication to your project using built-in Django authentication.
`path("accounts/", include("django.contrib.auth.urls"))`
After that you should delete accounts app from project